plex/umbrel-app.yml

```diff
@@ -26,17 +29,15 @@ torOnly: false
 permissions:
   - STORAGE_DOWNLOADS
 releaseNotes: >-
-  This update includes many bug fixes and performance improvements, as well as the following new features:
+  This update includes many bug fixes and performance improvements, as well as the following new changes:
   
-  - New “Deep Cuts” music station.
+
+  LinuxServer Changes:
+    - Install unrar from linuxserver repo.
+
+  Remote Changes:
+    - Data change at json endpoint https://plex.tv/api/downloads/5.json
   
-  - Hardware transcoding is now supported for SDR media on AMD GPUs on Linux
-  
-  - Hardware transcoding is now supported when running Plex Media Server on headless Windows systems
-  
-  - The GPU used for transcoding can now be selected in server preferences
-  
-  
-  See full release notes here: https://www.videohelp.com/software/Plex/version-history
+  See full release notes here: https://github.com/linuxserver/docker-plex/releases
```

Comment:
I think these release notes are related to the docker image specifically, and don't actually describe any changes to Plex Media Server. If you scroll back through all the Github releases they all look pretty much identical.

I've been using release notes from here: https://www.videohelp.com/software/Plex

Reply:
Ahh right, I've updated them.